ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/ContractsReader.scala

```diff
@@ -344,16 +359,24 @@ private[appendonlydao] sealed class ContractsReader(
            """
   }
 
-  private def lookupContractKeyQuery(readers: Set[Party], key: Key): SimpleSql[Row] = {
+  private def lookupContractKeyQuery(
+      readers: Set[Party],
+      key: Key,
+      // TODO Remove SQL references to `parameters` table when removing the `ledgerEndSequentialId` parameter optionality
```

Comment:
```suggestion
      // TODO append-only: Remove SQL references to `parameters` table when removing the `ledgerEndSequentialId` parameter optionality
```

same change at other places